includes/import-products.php

```diff
@@ -139,10 +142,30 @@ function dw_add_woocommerce_product($course) {
 		return ($startA < $startB) ? -1 : 1;
 
 	});
+
+	dw_format_and_save_dates($product, $startDate, $courseParts);
+
+	update_post_meta($product->get_id(), '_product_attributes', $attributes);
+
+	return $product;
+}
+
+/**
+ * @param WC_Product $product
+ * @param DateTime $date
+ * @param array $courseParts
+ *
+ * @throws Exception
+ */
+function dw_format_and_save_dates(WC_Product $product, DateTime $date, array $courseParts) {
```

Comment:
@Dirkhuethorst, we don't have tests for functions like this?

Reply:
Not really, since it we have to mock the weird `wp_set_object_terms` functions to make that work, and I don't see an easy way

Reply:
Too much hassle Exception